freqtrade/persistence.py

```diff
@@ -68,66 +70,86 @@ def get_column_def(columns, column: str, default: str) -> str:
     return default if not has_column(columns, column) else column
 
 
-def check_migrate(engine) -> None:
+def check_migrate(engine) -> bool:
     """
     Checks if migration is necessary and migrates if necessary
     """
     inspector = inspect(engine)
 
+    cols = inspector.get_columns('trades')
+
+    needs_migration = False
+
+    if not has_column(cols, 'bot_id'):
+        engine.execute("alter table trades add bot_id integer")
```

Comment:
any reason you're doing this column specific migration and not relying on sqlalchemy to recreate the table and simply update the insert statement with a default-value for the new column?

Reply:
oh right.. this is not needed since migration recreates the schema `_DECL_BASE.metadata.create_all(engine)`

Thanks